lib/service/service.go

```diff
@@ -753,9 +754,7 @@ func (process *TeleportProcess) getInstanceRoleEventMapping() map[types.SystemRo
 	process.Lock()
 	defer process.Unlock()
 	out := make(map[types.SystemRole]string, len(process.instanceRoles))
-	for role, event := range process.instanceRoles {
-		out[role] = event
-	}
+	maps.Copy(out, process.instanceRoles)
```

Comment:
nit: may be better to use maps.Clone here.

Reply:
This won't work. `maps.Clone` will result in a nil output given nil input, but the code here always results in a non-nil `out` map.